docs/source/upcoming_release_notes/385-ECS-5104_Hutch-python_feature:_auto-close_after_timeout.rst

```diff
@@ -0,0 +1,22 @@
+385 ECS-5104 Hutch-python feature: auto-close after timeout
+#################
+
+API Changes
+-----------
+- Created an IPython extension that launches a daemon thread that runs an instance of IPythonSessionTimer class object.
```

Comment:
We usually use this section to denote breaking API changes, which I wouldn't classify this as.  Nothing you've done here breaks hutch-python's interactions with other packages / interfaces, since we use a default value if nothing in the user's hutch-python configuration changes

Reply:
Ah ok, I removed this line.

Reply:
I just realized how many minor commits I have in my features branch (mostly fixing errors). Are those commits squashed when merging the pull request?

Reply:
We can squash before merging.  I usually don't mind having all the commits but if there's a lot of back and forth we can